apps/site/lib/site_web/controllers/schedule/line.ex

```diff
@@ -111,17 +111,31 @@ defmodule SiteWeb.ScheduleController.Line do
     variant = schedule_direction["variant"]
     expanded = Map.get(conn.query_params, "expanded")
     reverse_direction_id = reverse_direction(direction_id)
-    route_shapes = LineHelpers.get_route_shapes(route.id, direction_id)
     route_stops = LineHelpers.get_route_stops(route.id, direction_id, deps.stops_by_route_fn)
+    # Both line.ex and helpers.ex have this function defined. Redundant?
```

Comment:
If there's indeed a redundancy, could we refactor?

Reply:
Maybe could we add a tech debt ticket for it?  I left one or two other "todo" type comments, but since I didn't _need_ to refactor it to make it work, I left the redundancy as-is.  (I didn't add the redundancy in this ticket - it was there from previously)